gradle/scripts/lib/java-publish.gradle

```diff
@@ -69,10 +56,10 @@ configure(projectsWithFlags('publish', 'java')) {
                             }
                         }
 
-                        // Strip out shaded dependencies. We manually create shading tasks instead of applying
-                        // the shadow plugin itself so need to do it ourselves.
                         if (rootNode.get('dependencies')) {
                             def dependencies = rootNode.get('dependencies')[0]
```

Comment:
So by default, is `dependencies` what we defined in the `dependencies` section in gradle? or is it the resolved versions? I'm assuming the former since L11-L23 has been deleted, but just wanted to make sure.

Reply:
Before removing L11~L23, there were resolved dependencies. After removing the lines, they are unresolved dependencies defined in `build.gradle`.